assets/src/edit-story/elements/image/editCropMovable.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Movable from '../../components/movable';
+import { getFocalFromOffset } from './util';
+
+function EditCropMovable( {
+	setProperties, cropBox, croppedImage,
+	x, y,
+	offsetX, offsetY, imgWidth, imgHeight,
+} ) {
+	const moveableRef = useRef();
+	const cropRef = useRef( [ 0, 0, 0, 0 ] );
+
+	// Refresh moveables to ensure that the selection rect is always correct.
+	useEffect( () => {
+		if ( moveableRef.current ) {
+			moveableRef.current.updateRect();
+		}
+	} );
```

Comment:
This could be rewritten to using `useLayoutEffect` - then you can remove the `if` as it's always run after the component has been rendered. But I'd like to see some dependencies - should it really be done on every render or only if something specific changes?

Reply:
Removed `if`s. The `useEffect` works fine with this as well. It's actually a leftover from the time when these were combined together in a single module and were on/off selectively. Now that on/off control is done by the main component, so the refs can no longer be null.

Unfortunately, we've found that `updateRect()` is needed very often. On the plus side, however, it's very cheap if nothing really changed.